packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -72,6 +77,12 @@ export class Material extends ThreeDOMElement implements MaterialInterface {
   private[$clearcoatTexture]!: TextureInfo;
   private[$clearcoatRoughnessTexture]!: TextureInfo;
   private[$clearcoatNormalTexture]!: TextureInfo;
+  private[$sheenColorTexture]!: TextureInfo;
+  private[$sheenRoughnessTexture]!: TextureInfo;
+  private[$transmissionTexture]!: TextureInfo;
+  private[$thicknessTexture]!: TextureInfo;
+  private[$specularTexture]!: TextureInfo;
+  private[$specularColorTexture]!: TextureInfo;
```

Comment:
I really hate symbols, but they're only necessary directly in a publicly-accessible object. Maybe we can assemble these into a single object and just have one symbol instead?

Reply:
Maybe this map approach is better?